text/0000-volta-use.md

```diff
@@ -0,0 +1,103 @@
+- Feature Name: volta_use
+- Start Date: 2024-02-22
+- RFC PR: (leave this empty)
+- Volta Issue: (leave this empty)
+
+# Summary
+
+[summary]: #summary
+
+Adding a new command, `volta use`, which combines the effects of `volta install` and `volta pin` into a single command that is more intuitive and better matches the mental model of Volta.
+
+# Motivation
+
+[motivation]: #motivation
+
+Early on in Volta's history, we picked the verb `install` to represent choosing the default version of tools (i.e. the version used when outside of a pinned project). This was chosen, in part, to align with users' expectations from other similar tools. However, due to Volta's auto-download behavior when it detects a version that isn't currently cached, `install` centers the wrong action. The more important part of `volta install` is the selecting a default version.
+
+This has led to some confusion for users. We get questions fairly regularly asking how users should select an "active" version of Node, once they have installed them using `volta install`. Additionally, by centering the install action, rather than the selection, `volta install` doesn't help users internalize Volta's mental model of "select a version and Volta will automatically download and make sure it's available."
+
+Selecting a different verb for the action of choosing a tool version should reduce confusion and make it more clear how Volta is intended to work.
+
+# Pedagogy
+
+[pedagogy]: #pedagogy
+
+This will be a fairly large change to our recommended workflows. As `volta use` will work for both pinning _and_ selecting a default version, we will need to update the documentation throughout to take advantage of the new command. We will also likely want to write a blog post for the release which includes this change, explaining the reasoning and the new workflows.
+
+Notably, we will _not_ be removing the existing `volta install` and `volta pin` commands right away. They will continue to work as they already do. In the future, we might consider adding notifications to them to point users to the updated command, but for the initial implementation we don't want to disrupt current users' workflows.
+
+All that said, the new command should simplify things a lot, especially for new users. It will allow us to explain Volta by saying that you pick a version with `volta use`, and don't worry about installing it: Volta will handle making it available. It also should do a better job of living up to the claim on our website by truly having a _single_ command that users can run to get set up.
+
+# Details
+
+[details]: #details
+
+## Command definition
+
+```
+USAGE:
+    volta use [FLAGS] <tool[@version]>...
+
+FLAGS:
+    -g, --global    Select the global version of a tool, which is used everywhere outside of pinned projects
```

Comment:
Historically, at least, we have always used the language of “defaults” rather than “globals” to distinguish that Volta’s mental model is a bit different.

```suggestion
    -d, --default    Select the default version of a tool, which is used everywhere outside of pinned projects
```